Simplified/it.lproj/Localizable.strings

```diff
@@ -263,6 +263,7 @@
 "Please wait..." = "Attendere, prego...";
 "Delete all the bookmarks you have saved in the cloud." = "Tutti i segnalibri salvati sul server saranno rimossi.";
 "You have successfully deleted your SimplyE data" = "I tuoi dati di SimplyE sono stati rimossi";
+"You have successfully deleted your Open eBooks data" = "I tuoi dati di Open eBooks sono stati rimossi";
```

Comment:
@ettore Can you check if the Italian wording looks right to you?

Reply:
actually these strings do look bit odd.... can you change both OE and SE strings to say: "I dati del tuo utente di SimplyE / Open eBooks sono stati rimossi"

Reply:
also at line 260 / 261 let's just do simply for both apps: "Rimuovi i miei dati". It's implicit we're talking about SE/OE, but if I add SimplyE / OpenE it always sounds weird in Italian. (Italian doesn't have the Saxon genitive form, so I'd be forced to use the equivalent of `of`, and that gets weird because we're referring to "my" data, but also the data "of" SimplyE/OE....)